NBitcoin/BlockrTransactionRepository.cs

```diff
@@ -125,7 +125,7 @@ public Task PutAsync(uint256 txId, Transaction tx)
 				var json = JObject.Parse(result);
 				var status = json["status"];
 				var code = json["code"];
-				if(status != null && status.ToString() == "error")
+				if(status != null && (status.ToString() == "error" || status.ToString() == "fail")

```

Comment:
Right. I've verified this:
```bash
~# curl -d '{"hex":"BAD-TX-HEX"}' http://btc.blockr.io/api/v1/tx/push
{"status":"fail","data":"Could not decode your transaction!","code":500,"message":""}
```

Then, you should remove the `status.ToString() == "error"` part because it is wrong.